sync-stdlib-doc.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+# Written in [Amber](https://amber-lang.com/)
+# version: 0.3.5-alpha
+# date: 2024-09-05 11:54:56
+file_exist__1_v0() {
```

Comment:
Why do we need this script?

Reply:
I saw that is the standard to provide the amber script already compiled to bash